guides/release/testing/testing-components.md

```diff
@@ -156,23 +156,23 @@ Imagine you have the following component that changes its title when a button is
 > component magic-title`.
 
 ```javascript {data-filename="app/components/magic-title.js"}
-import Component from '@ember/component';
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
 
-export default Component.extend({
-  title: 'Hello World',
+export default class MagicTitleComponent extends Component {
+  title = 'Hello World';
 
-  actions: {
-    updateTitle() {
-      this.set('title', 'This is Magic');
-    }
-  }
-});
+  @action
+  updateTitle() {
+    this.set('title', 'This is Magic');
+  };
+};
 ```
 
 ```handlebars {data-filename="app/templates/components/magic-title.hbs"}
 <h2>{{this.title}}</h2>
 
-<button class="title-button" {{action "updateTitle"}}>
+<button class="title-button" onclick={{action this.updateTitle}}>
```

Comment:
We are avoiding `onclick=` usage (see @mixonic's awesome explanation over at https://github.com/ember-learn/guides-source/pull/598#discussion_r265390517)

```suggestion
<button class="title-button" {{action this.updateTitle}}>
```